aspnetcore/fundamentals/logging/loggermessage.md

```diff
@@ -17,45 +16,63 @@ By [Luke Latham](https://github.com/guardrex)
 
 [View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/fundamentals/logging/loggermessage/sample/) ([how to download](xref:tutorials/index#how-to-download-a-sample))
 
-The sample app demonstrates `LoggerMessage` features with a simple quote tracking database. The user adds, deletes, and deletes all quotes from an in-memory database. As these operations occur, log messages are generated using static extension methods that trigger logging `Action`s.
+The sample app demonstrates `LoggerMessage` features with a basic quote tracking database. The app adds, deletes single, and deletes all quotes from an in-memory database. As these operations occur, log messages are generated using static extension methods that trigger logging actions.
 
 ## LoggerMessage.Define
 
 [Define(LogLevel, EventId, String)](/dotnet/api/microsoft.extensions.logging.loggermessage.define) creates an `Action` delegate for logging a message. `Define` overloads permit passing up to six type parameters to a named format string (template).
 
-The string provided to the `Define` method is a template and not an interpolated string. Placeholders are filled in the order that the types are specified. Placeholder names in the template should be descriptive and consistent across templates because they serve as property names within structured log data. The sample app uses and we recommend [Pascal casing](/dotnet/standard/design-guidelines/capitalization-conventions) for placeholder names (for example, "{Count}", "{FirstName}").
-
 ## LoggerMessage.DefineScope
 
 [DefineScope(String)](/dotnet/api/microsoft.extensions.logging.loggermessage.definescope) creates a `Func` delegate for defining a [log scope](xref:fundamentals/logging/index#log-scopes). `DefineScope` overloads permit passing up to three type parameters to a named format string (template).
 
-The string provided to the `DefineScope` method is a template and not an interpolated string. Placeholders are filled in the order that the types are specified. Placeholder names in the template should be descriptive and consistent across templates because they serve as property names within structured log data. The sample app uses and we recommend [Pascal casing](/dotnet/standard/design-guidelines/capitalization-conventions) for placeholder names (for example, "{Count}", "{FirstName}").
+## Define and DefineScope
+
+The string provided to the `Define` and `DefineScope` methods is a template and not an interpolated string. Placeholders are filled in the order that the types are specified. Placeholder names in the template should be descriptive and consistent across templates.  Placeholder names should be descriptive and consistent because they serve as property names within structured log data. We recommend [Pascal casing](/dotnet/standard/design-guidelines/capitalization-conventions) for placeholder names. For example, `"{Count}", "{FirstName}"`.
```

Comment:
Actually, the placeholders would each get inline code styling without the quotes if setup this way ...

For example, `{Count}`, `{FirstName}`.

There's also a double-space after the 3rd sentence.